bin/10_4/10_2_modified/src/Store_temps.cpp

```diff
@@ -0,0 +1,94 @@
+#include <cstdint>
+#include <exception>
+#include <fstream>
+#include <iostream>
+#include <vector>
+
+constexpr std::uint64_t number_of_readings = { 5 };
+
+struct Reading {
+    int hour;
```

Comment:
The order of members should be
    int hour;
    char scale;
    double temperature;
to reduce the size of padding